src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,37 @@
+package nextstep.ladder.domain;
+
+import nextstep.ladder.domain.exceptions.InvalidLadderHeightException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Ladder {
+    private static final int MIN_HEIGHT = 1;
+    private List<Line> lines;
+
+    private Ladder(List<Line> lines) {
+        this.lines = new ArrayList<>(lines);
+    }
+
+    public static Ladder create(int ladderHeight, int countOfPerson, PointAddStrategy pointAddStrategy) {
+        ladderHeightValidation(ladderHeight);
+        List<Line> lines = new ArrayList<>();
+        IntStream.range(0, ladderHeight).forEach(num -> lines.add(Line.create(countOfPerson, pointAddStrategy)));
```

Comment:
가독성을 위해 적당한 개행하면 좋을 것 같아요:)
```suggestion
        IntStream.range(0, ladderHeight)
                        .forEach(num -> lines.add(Line.create(countOfPerson, pointAddStrategy)));
```